### PR TITLE
Avoid conflict with the escape key and general popup

### PR DESCRIPTION
**Pull request by PascalPiche**
Change the open trigger of the dev panel to be shift + Pause/Break and keep escape also to close it. Dont use escape key to open de panel.

### CODE REVIEW

File: workspace/assets/js/dev/panel.js [hunk 1]
```diff
@@ -158,10 +158,15 @@
...
 		});
+
 		var attached = false;
 
 		$(document).on('keydown', function (e) {
-			if (e.which === global.keys.escape) {
+			if (e.which === global.keys.escape && attached) {
+				var attFx = attached ? 'detach' : 'appendTo';
+				panel[attFx](attached ? undefined : body);
+				attached = !attached;
```
Comment (nitriques) [inline workspace/assets/js/dev/panel.js]: I would make it
```suggestion
				panel.detach();
				attached = false;
```

File: workspace/assets/js/dev/panel.js [hunk 2]
```diff
@@ -158,10 +158,15 @@
...
+			} else if (e.which === global.keys.pause_break && e.shiftKey) {
 				var attFx = attached ? 'detach' : 'appendTo';
 				panel[attFx](attached ? undefined : body);
 				attached = !attached;
```
Comment (nitriques) [inline workspace/assets/js/dev/panel.js]: Same here.

Comment (PascalPiche) [reply workspace/assets/js/dev/panel.js]: Its the toggle part. may open or close the popup...

File: workspace/assets/js/dev/panel.js (hunk 1 above)
Comment (PascalPiche) [reply workspace/assets/js/dev/panel.js]: Fixed

File: workspace/assets/js/dev/panel.js (hunk 2 above)
Comment (nitriques) [reply workspace/assets/js/dev/panel.js]: ah ok, nvm.

**Pull request merged**